application/src/main/java/org/thingsboard/server/service/telemetry/DefaultAlarmSubscriptionService.java

```diff
@@ -144,7 +121,11 @@ public Alarm createOrUpdateAlarm(Alarm alarm) {
                         .type(AlarmCommentType.SYSTEM)
                         .comment(JacksonUtil.newObjectNode().put("text", String.format("Alarm severity was updated from %s to %s", oldSeverity, result.getAlarm().getSeverity())))
                         .build();
-                alarmCommentService.createOrUpdateAlarmComment(alarm.getTenantId(), alarmComment);
+                try {
+                    alarmCommentService.saveAlarmComment(alarm, alarmComment, null);
+                } catch (ThingsboardException e) {
+                    log.error("Failed to save alarm comment", e);
```

Comment:
Is this something for debug our you really need this?

Reply:
ThingsboardException is checked, didn't want to add it to the signature